frame/support/test/tests/pallet.rs

```diff
@@ -1499,6 +1609,13 @@ fn test_storage_info() {
 				max_values: Some(11),
 				max_size: Some(14 + 8 + 16),
 			},
+			StorageInfo {
+				pallet_name: b"Example".to_vec(),
+				storage_name: b"NMap3".to_vec(),
+				prefix: prefix(b"Example", b"NMap3").to_vec(),
+				max_values: None,
+				max_size: Some(7 + 4 + 32),
```

Comment:
I don't actually know how to get these numbers, I'm just making educated guesses on what they are.

Reply:
@ggwpez @kianenigma @shawntabrizi Did we ever figure out how these numbers are calculated?

Reply:
Found out the formula, it's actually pretty easy to figure out, it's just that we've been making up random numbers in this test.